core/chains/evm/config/v2/defaults/Harmony_Testnet.toml

```diff
@@ -0,0 +1,5 @@
+ChainID = '1666700000'
+LinkContractAddress = '0x8b12Ac23BFe11cAb03a634C1F117D64a7f2cFD3e'
+GasPriceDefault = '5 gwei'
+MinIncomingConfirmations = 1
+LogPollInterval = '2s'
```

Comment:
We have a few variables relating to average block time... I wonder if we want to create just one "average block time" variable and calculate the others from it?

e.g. `nodeDeadAfterNoNewHeadersThreshold`, `blockEmissionIdleWarningThreshold`, `samplingInterval` etc

Reply:
We should probably create a follow up story for this one. Do you have anything else to add?
FWIW I am in favor of consolidating and deriving as much as we can, but I wonder if it could be too limiting to _remove_ the separate fields entirely, and if they should be kept around to be overridden individually if necessary. In that case, this change would be more about wiring and documenting the non-static default logic in a standard way.